packages/modules/pgr/src/components/FormComposer.js

```diff
@@ -34,7 +34,7 @@ export const FormComposer = (props) => {
           </div>
         );
       case "textarea":
-        return <TextArea className="field" {...populators} inputRef={register(populators.validation)} />;
+        return <TextArea className="field" name={""} {...populators} inputRef={register(populators.validation)} />;
```

Comment:
put, `populators.name || ""` in all these where name is req

Reply:
some places we dont have populators
```
<TextArea value={comment} onChange={(e) => setComment(e.target.value)} name={""} />
```